packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.html

```diff
@@ -111,31 +111,6 @@
                 data-translate="{{ 'dedicated_server_interfaces_type_' + $value + '_label' }}"
             ></span>
         </oui-datagrid-column>
-        <oui-datagrid-column
-            data-title=":: 'dedicated_server_interfaces_ola_column_network_offer' | translate"
```

Comment:
**thought:** Additional translation files can be cleaned up I think.
Could be done via a separate request.